arm-sql/2014-04-01/swagger/capabilities.json

```diff
@@ -0,0 +1,319 @@
+        "version": "2014-04-01"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Sql/locations/{locationName}/capabilities": {
+            "get": {
+                "tags": [
+                    "Capabilities"
+                ],
+                "operationId": "Capabilities_Get",
+                "description": "Returns information about the Azure SQL capabilities available for the specified region.",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "name": "locationName",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "The name of the region for which the Azure SQL capabilities are retrieved."
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "",
+                        "schema": {
+                            "$ref": "#/definitions/LocationCapability"
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "MaxSizeCapability": {
+            "properties": {
+                "limit": {
+                    "readOnly": true,
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "Gets the maximum size of the database (see 'unit' for the units)."
+                },
+                "unit": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Gets the units that the limit is expressed in.",
+                    "enum": [
+                        "Megabytes",
+                        "Gigabytes",
+                        "Terabytes",
+                        "Petabytes"
+                    ],
+                    "x-ms-enum": {
+                        "name": "MaxSizeUnits",
+                        "modelAsString": true
+                    }
+                },
+                "status": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Gets the status of the maximum size capability.",
+                    "enum": [
+                        "Visible",
+                        "Available",
+                        "Default"
+                    ],
+                    "x-ms-enum": {
+                        "name": "CapabilityStatus",
+                        "modelAsString": true
```

Comment:
Do you really want to model this as a string and not an enum? Do you expect new values to appear without bumping the api version?